### PR TITLE
added ci files

### PR DESCRIPTION
**Pull request by daiLlew**
Added CI config.

### CODE REVIEW

File: ci/audit.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-java
+
+inputs:
+  - name: restolino
+
+run:
+  path: restolino/ci/scripts/audit.sh
```
Comment (morses-code) [inline ci/audit.yml:13]: New line

File: ci/build.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    repository: onsdigital/dp-concourse-tools-java
+
+inputs:
+  - name: restolino
+
+outputs:
+  - name: build
+
+run:
+  path: restolino/ci/scripts/build.sh
```
Comment (morses-code) [inline ci/build.yml:16]: New line

File: ci/lint.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-java
+
+inputs:
+  - name: restolino
+
+run:
+  path: restolino/ci/scripts/lint.sh
```
Comment (morses-code) [inline ci/lint.yml:13]: New line

File: ci/scripts/audit.sh [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+pushd restolino
+    make audit
+popd
```
Comment (morses-code) [inline ci/scripts/audit.sh:5]: New line

File: ci/scripts/build.sh [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+pushd restolino
+    make build
+popd
```
Comment (morses-code) [inline ci/scripts/build.sh:5]: New line

File: ci/scripts/lint.sh [hunk 6]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -eux
+
+pushd restolino
+    make lint
+popd
```
Comment (morses-code) [inline ci/scripts/lint.sh:5]: New line

File: ci/unit.yml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+image_resource:
+  type: docker-image
+  source:
+    repository: onsdigital/dp-concourse-tools-java
+
+inputs:
+  - name: restolino
+
+run:
+  path: restolino/ci/scripts/unit.sh
```
Comment (morses-code) [inline ci/unit.yml:13]: New line

File: Makefile [hunk 8]
```diff
@@ -1,15 +1,19 @@
...
-	mvn -Dmaven.test.skip -Dossindex.skip=true clean package
+	mvn clean package -Dmaven.test.skip -Dossindex.skip=true
 
 .PHONY: test
 test:
 	mvn -Dossindex.skip=true test
 
+.PHONY: lint
+lint:
+	exit
```
Comment (morses-code) [inline Makefile:18]: Any point adding this?

**Review by morses-code — CHANGES_REQUESTED** [review]
Few picky comments

File: ci/audit.yml (hunk 1 above)
Comment (daiLlew) [reply ci/audit.yml:13]: What is the technical requirement/rational behind requiring a new line at the end of this file?

File: Makefile (hunk 8 above)
Comment (daiLlew) [reply Makefile:18]: The lint pipeline will exist regardless, adding this is essentially a nop impl to pacify the pipeline.

**Review by morses-code — APPROVED** [review]

**Pull request merged**